src/ring_theory/polynomial/chebyshev/dickson.lean

```diff
@@ -0,0 +1,88 @@
+/-
+Copyright (c) 2021 Julian Kuelshammer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Julian Kuelshammer
+-/
+
+import data.polynomial.eval
+import ring_theory.polynomial.chebyshev.defs
+
+/-!
+# Dickson polynomials
+
+The (generalised) Dickson polynomials are a family of polynomials indexed by `ℕ × ℕ`,
+with coefficients in a commutative ring `R` depending on an element `a∈R`.
```

Comment:
It would be good to state the definition explicitly here. That will also make the role of `a` more clear.